game-core/src/main/java/games/strategy/triplea/delegate/RocketsFireHelper.java

```diff
@@ -103,14 +100,12 @@ private static void fireWW2V2(
       // Ask the user where each rocket launcher should target.
```

Comment:
This comment was added in #1455.  Is it still applicable after this revert?

Reply:
Arguably it is. The user is asked where the rocket should target still. It just doesn't store the target - it fires it immediately. I'd be ok with it being left or being removed.